general/error.hpp

```diff
@@ -91,6 +95,17 @@ void mfem_warning(const char *msg = NULL);
       else                                                              \
          mfem::mfem_warning(mfemMsgStream.str().c_str());               \
    }
+#else
+#define _MFEM_MESSAGE(msg, warn)                                        \
+   {                                                                    \
+      static std::ostringstream mfemMsgStream;                          \
+      mfemMsgStream << std::setprecision(16);                           \
+      mfemMsgStream << std::setiosflags(std::ios_base::scientific);     \
+      mfemMsgStream << msg << MFEM_LOCATION;                            \
+      if (mfemMsgStream.str().c_str())                                  \
+         std::cout << mfemMsgStream.str().c_str() << std::endl;         \
```

Comment:
Since this is no longer calling into mfem_error/mfem_warning we will no longer stop in when we hit an MFEM_ERROR.  Were you have trouble getting the JIT stuff to work with the exceptions and then libunwind stuff?

Reply:
As the JITed code is not linked with MFEM's library, we cannot use `mfem::mfem_error` or `mfem::mfem_warning`.
This allows to use the header-only capability, but needs this last change to avoid unresolved symbols.
I agree we will not be able to set the breakpoint on `mfem::mfem_error` in JITed code.

Reply:
Hmm, might it be possible to keep the old behavior for non-jitted functions?

Reply:
It is the case: the `#ifndef MFEM_JIT_COMPILATION` block is only for JITed code.
The `MFEM` code behavior is not changed, only the JITed code will use `std::cout` instead of `mfem::mfem_error`/`mfem::mfem_warning`.